runtime/module.go

```diff
@@ -176,6 +157,35 @@ func SetupAppBuilder(inputs AppInputs) {
 	}
 }
 
+func ProvideInterfaceRegistry(customGetSigners []signing.CustomGetSigner) (codectypes.InterfaceRegistry, error) {
+	signingOptions := signing.Options{
+		// using the global prefixes is a temporary solution until we refactor this
+		// to get the address.Codec's from the container
+		AddressCodec: address.Bech32Codec{
+			Bech32Prefix: sdk.GetConfig().GetBech32AccountAddrPrefix(),
+		},
+		ValidatorAddressCodec: address.Bech32Codec{
+			Bech32Prefix: sdk.GetConfig().GetBech32ValidatorAddrPrefix(),
+		},
+	}
+	for _, signer := range customGetSigners {
+		signingOptions.DefineCustomGetSigners(signer.MsgType, signer.Fn)
```

Comment:
This code does nothing I'm pretty sure. The receiver is a struct not a pointer

Reply:
This code does something, registers a custom signer fn. If you'd like to prove to this yourself, comment it out and run `TestDefineCustomGetSigners` in the x/tx integration tests.

Reply:
Because map is a pointer type?

Reply:
Yea. I'm doing an iteration and copy of customGetSignerFuncs in in construction of signing.Context so that changing options after construction won't alter context.